ext/bg/js/backend.js

```diff
@@ -809,6 +811,11 @@ class Backend {
         return portName;
     }
 
+    async _onApiImportDictionaryArchive({archiveContent, details}, sender, onProgress) {
+        this._validatePrivilegedMessageSender(sender);
```

Comment:
Should this be moved to the handler info object like in https://github.com/FooSoft/yomichan/pull/497 (separately)?

Reply:
Probably, now that there's somewhat of a convention for it.